webview-ui/src/components/mcp/marketplace/McpSubmitCard.tsx

```diff
@@ -0,0 +1,51 @@
+const McpSubmitCard = () => {
+	return (
+		<div
+			style={{
+				display: "flex",
+				flexDirection: "column",
+				alignItems: "center",
+				gap: "24px",
+				padding: "32px 20px",
+				marginTop: "16px",
+			}}>
+			{/* Logo */}
+			<img
+				src="https://storage.googleapis.com/cline_public_images/cline.png"
+				alt="Cline bot logo"
+				style={{
+					width: 64,
+					height: 64,
+					borderRadius: 8,
+				}}
+			/>
```

Comment:
we have integrated tailwind and shadcn
https://ui.shadcn.com/ 

it will be cool if you you can leverage it 💪,